emperor/support_files/js/view-controller.js

```diff
@@ -353,12 +376,52 @@ define([
       columns.unshift(options.slickGridColumn);
     }
 
+    var dataView = new Slick.Data.DataView(), searchString = '';
```

Comment:
Does this library need an entry in https://github.com/biocore/emperor/blob/new-api/LICENSE.md ?

Reply:
It does not, the module is part of the slickgrid library, which is already in use. 👍